ts/components/MessageComponent.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+import { Icon, Left, ListItem, Right, Text } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import { convertDateToWordDistance } from "../utils/convertDateToWordDistance";
+
+export type OwnProps = Readonly<{
+  serviceOrganizationName: string;
+  serviceDepartmentName: string;
+  subject: string;
+  key: string;
+  date: Date;
```

Comment:
Is this `date` the `created_at` attribute?

Reply:
Fixed.